<ui version="4.0" >
 <class>WebServicesPropertyDialog</class>
 <widget class="QDialog" name="WebServicesPropertyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WebServices Property</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Web Services</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QComboBox" name="m_serviceNameComboBox" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>&amp;Service :</string>
        </property>
        <property name="buddy" >
         <cstring>m_serviceNameComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="m_serviceTypeComboBox" >
        <item>
         <property name="text" >
          <string>CRUD</string>
         </property>
         <property name="icon" >
          <iconset resource="../images/application.qrc" >:/exec.png</iconset>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>API</string>
         </property>
         <property name="icon" >
          <iconset resource="../images/application.qrc" >:/dbgmemview.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>WebServices &amp;Type :</string>
        </property>
        <property name="buddy" >
         <cstring>m_serviceTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="m_webServicesNameComboBox" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>WebServices &amp;Name :</string>
        </property>
        <property name="buddy" >
         <cstring>m_webServicesNameComboBox</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Context</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="2" >
           <widget class="QLineEdit" name="m_passwordLineEdit" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2" >
           <widget class="QLineEdit" name="m_targetLineEdit" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>&amp;Password :</string>
            </property>
            <property name="buddy" >
             <cstring>m_passwordLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>&amp;User :</string>
            </property>
            <property name="buddy" >
             <cstring>m_entityLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Ta&amp;rget :</string>
            </property>
            <property name="buddy" >
             <cstring>m_targetLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>&amp;Entity : </string>
            </property>
            <property name="buddy" >
             <cstring>m_entityLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLineEdit" name="m_userLineEdit" />
          </item>
          <item row="0" column="2" >
           <widget class="QLineEdit" name="m_entityLineEdit" >
            <property name="inputMask" >
             <string>#999; </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>CRUD</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>View&amp;Object Name :</string>
              </property>
              <property name="buddy" >
               <cstring>m_viewObjectLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_viewObjectLineEdit" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Retrieve &amp;Key :</string>
              </property>
              <property name="buddy" >
               <cstring>m_retrieveKeyLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_retrieveKeyLineEdit" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QTableWidget" name="m_paramTableWidget" >
              <row>
               <property name="text" >
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text" >
                <string>Param Value</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QToolButton" name="m_addParamToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/ajouter.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="m_delParamToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/supprimer.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="m_previousParamToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/previous.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="m_nextParamToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/next.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Update/Create :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QTableWidget" name="m_setTableWidget" >
              <row>
               <property name="text" >
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text" >
                <string>Field Name</string>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>Field Value</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QToolButton" name="m_addFieldToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/ajouter.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="m_delFieldToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/supprimer.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="m_previousFieldToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/previous.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="m_nextFieldToolButton" >
                <property name="text" >
                 <string>...</string>
                </property>
                <property name="icon" >
                 <iconset resource="../images/application.qrc" >:/next.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>API</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QTreeView" name="treeView" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_testButton" >
       <property name="text" >
        <string>Test ...</string>
       </property>
       <property name="icon" >
        <iconset resource="../images/application.qrc" >:/exec.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_webServicesNameComboBox</tabstop>
  <tabstop>m_serviceNameComboBox</tabstop>
  <tabstop>m_serviceTypeComboBox</tabstop>
  <tabstop>m_entityLineEdit</tabstop>
  <tabstop>m_userLineEdit</tabstop>
  <tabstop>m_passwordLineEdit</tabstop>
  <tabstop>m_targetLineEdit</tabstop>
  <tabstop>m_retrieveKeyLineEdit</tabstop>
  <tabstop>m_paramTableWidget</tabstop>
  <tabstop>m_setTableWidget</tabstop>
  <tabstop>m_testButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources>
  <include location="../images/application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>WebServicesPropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>531</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WebServicesPropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>612</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_serviceTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>342</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>48</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
